analysis/scripts/2233-power-calculations.Rmd

```diff
@@ -40,6 +40,17 @@ library(ggthemes)
 library(DesignLibrary)
 library(MASS)
 library(flextable)
+library(coin)
+library(estimatr)
+library(furrr)
+library(future)
+```
+
+```{r parse_params}
+## In case you are not using RStudio but need the params
```

Comment:
I prefer to use editors other than RStudio so I figured out how to grab the params.